### ISSUE TITLE
Inspiration

### ISSUE THREAD

**Issue by seankross** [datacamp/stats-apps#3]
You probably know about this site already, if not it's worth taking a look: http://www.rossmanchance.com/applets/

**Comment by ncarchedi** [comment]
@seankross Yup, but I find them to be horribly confusing. For example, this one: http://www.rossmanchance.com/applets/ChiSqShuffle.html?dolphins=1&count=1

No clear call to action for the student. Unclear what's input and what's output.

**Comment by seankross** [comment]
I 100% agree! This could be a way better version.

**Comment by ncarchedi** [comment]
@seankross Would love to get your thoughts when I'm a bit further along with some of these. Working on a better version of the link above as I write this.
